Misc/NEWS.d/next/Security/2019-03-31-15-06-35.bpo-36495.ahXWSI.rst

```diff
@@ -0,0 +1 @@
+Fix two out-of-bound reads in the code that constructs abstract syntax trees.  Patch by Brad Larsen.
```

Comment:
I am not sure this fix needs a NEWS entry, especially in the Security section. The bug was introduces at the alpha stage, nobody should use it in production.

Reply:
I agree; I was originally going to leave that blank, but bedevere-bot said one was needed (or I didn't understand how to skip that check).

Reply:
Should I revert the commit that adds the NEWS entry?